src/state/selectors/sequences.js

```diff
@@ -16,11 +17,44 @@ export const getSequences = createSelector(
 
     if (v2TopRanges.length === 0 && topRangesOrRoot.length === 1) {
       v3RangeSequences = topRangesOrRoot[0].getRanges().filter(r => r.getBehavior() === 'sequence');
+
+      /**  Add manifesto canvases (items) to the ranges if 'items' property of ranges
+       *  is empty
+       */
+      if (v3RangeSequences.length > 0 && manifest.items && manifest.items.length > 0
+          && manifest.items[0].items && manifest.items[0].items.length > 0) {
+        /** Use manifesto canvases provided in manifest 'items' property to populate
+         * range 'items'/canvases
+        */
+        const canvases = manifest.items[0].items;
+
+        v3RangeSequences.map((sequence) => {
+          if (sequence.items.length === 0) {
+            const updatedSequence = sequence;
+            updatedSequence.items = sequence.canvases.map(canvasId => {
+              const fullCanvas = canvases.find(c => c.id === canvasId);
+              return fullCanvas;
+            });
+            return updatedSequence;
+          }
+          return sequence;
+        });
+      }
+    }
```

Comment:
Hi @mejackreed , I made a pull request to 'get' canvases in the iiif manifesto library as well so this specific change won't be needed in future and i will remove it later after that pull request gets accepted. The manifesto pull request is here https://github.com/IIIF-Commons/manifesto/pull/79. Also I was wondering if you got a chance to get feedback from the designer on whether the additional sequence (which includes all manifest canvases) can be created and displayed (even if its not listed as seqs). I can make the changes accordingly after feedback from the designer.